src/Domain/FunctionsOfTime/PiecewisePolynomial.hpp

```diff
@@ -36,8 +36,9 @@ class PiecewisePolynomial : public FunctionOfTime {
 
   auto get_clone() const noexcept -> std::unique_ptr<FunctionOfTime> override;
 
-  // NOLINTNEXTLINE(google-runtime-references)
-  WRAPPED_PUPable_decl_template(PiecewisePolynomial<MaxDeriv>);
+  // clang-tidy: google-runtime-references
```

Comment:
Just an FYI, I'll submit a PR that silences these warnings everyone in the code base from Charm++ automatically :)